frontend/src/routes/moods/thoughts/+page.server.js

```diff
@@ -19,12 +19,42 @@ export const load = async ({ serverLoadEvent, cookies }) => {
 
     const thoughts = response.data.data;
     const thoughtsDate = response.data.meta.date;
+    const records = [];
```

Comment:
Maybe it could be better with an array of json objects - not necessary to change, just for future reference (talking about `records` + `moodType`)